### DISCUSSION TITLE
remove weather section in waybar

### DISCUSSION THREAD

**Discussion by Linxford** [JaKooLit/Hyprland-Dots#258] [Q&A] [answered]
how  can i remove weather section in waybar ?

**Comment by Linxford** [comment]
Done!!! 

went to modify the waybar modules > custom/weather

**Comment by JaKooLit** [comment, marked as answer]
https://github.com/JaKooLit/Hyprland-Dots/wiki/TIPS#%EF%B8%8F-waybar-customizations

**Reply by Linxford** [reply]
thanks bro
